hugo/layouts/_default/list.algolia.json

```diff
@@ -19,53 +19,55 @@
     {{ $index := .index -}}
     {{ $skipWords := mul $cutoff $index }}
 
-    {{- $objectID := (print $page.File.UniqueID "_" (add $index 1)) -}}
-    {{- $title :=  $page.Title -}}
-    {{- $link := $page.RelPermalink -}}
-    {{- $publishDate := $page.PublishDate -}}
-    {{- $summary := $page.Summary -}}
-    {{- $content := "" -}}
-    {{- $tags := cond (isset $page.Params "tags") ($page.Params.tags) "" -}}
-    {{- $contentType := $page.FirstSection.Title -}}
+    {{- with $page.File -}}
+        {{- $objectID := (print $page.File.UniqueID "_" (add $index 1)) -}}
```

Comment:
would suggest to move this logic wiithin objectId
something like this
{{- $fileID := $page.File ? $page.File.UniqueID  : 'page' -}} > this is not valid code but just to give an idea
{{- $objectID :=  (print fileID "_" (add $index 1)) -}}